frontend/ui/widget/dictquicklookup.lua

```diff
@@ -751,7 +751,18 @@ function DictQuickLookup:getHtmlDictionaryCss()
         blockquote, dd {
             margin: 0 1em;
         }
+
+        ol, ul, menu {
+            margin: 0; padding: 0 1.7em;
```

Comment:
1.7? :-)

Reply:
Yep, with 1.5 I got "21..." truncated with various DPI settings:
![image](https://user-images.githubusercontent.com/24273478/129741493-3e128d0c-f5ad-4b8b-8f20-efffc4df98bd.png) ![image](https://user-images.githubusercontent.com/24273478/129741779-e45409a4-c62b-4d98-8124-d2575d82313d.png)

With 1.6 it looks ok, but 1.7 just to be sure it really is :)